test.html

```diff
@@ -106,7 +107,7 @@
   <label for="test_input">test input:</label><input id="test_input">
   <hr>
   <button class="big-ass-button" id="tat_button">Open TAT</button>
-  <button class="big-ass-button" id="copy_button">Copy shit</button>
+  <button class="big-ass-button" id="copy_button">Copy things</button>
```

Comment:
I felt uncomfortable about this...pls don't git blame though

Reply:
pfhahhahhahah! I love the class name,